optax/_src/alias.py

```diff
@@ -1488,7 +1488,63 @@ def sm3(
   parameters; 2) adapts the learning rates in an adaptive and data-driven manner
   (like Adagrad and unlike Adafactor); and 3) comes with rigorous convergence
   guarantees in stochastic convex optimization settings.
-  
+
+  The algorithm initializes with a learning rate :math:`\eta`, maintains 
+  cumulative squared gradients :math:`\mu_t(r)` for each weight component 
+  :math:`r`, and updates :math:`w_t` using received gradients :math:`g_t`, 
+  adjusting each component based on the minimum accumulated gradient 
+  :math:`\nu_t(i)`. It employs a convention where division by zero yields zero, 
+  optimizing convergence by adapting to varying gradients across different 
+  parts of the weight vector.
+
+  SM3-I Algorithm
+
+  .. math::
+    \begin{align*}
+    & \text{parameters: learning rate } \eta \\
+    & \text{initialize } w_1 = 0; \forall r \in [k]: \mu_0(r) = 0 \\
+    & \text{for } t = 1, \ldots, T \text{ do} \\
+    & \quad \text{receive gradient } g_t = \nabla \ell_t(w_t) \\
+    & \quad \text{for } r = 1, \ldots, k \text{ do} \\
+    & \quad \quad \mu_t(r) \leftarrow \mu_{t-1}(r) \\
+    & \quad \quad \quad \quad + \max_{j \in S_r} g_t^2(j) \\
+    & \quad \text{for } i = 1, \ldots, d \text{ do} \\
+    & \quad \quad \nu_t(i) \leftarrow \min_{r:S_r \ni i} \mu_t(r) \\
+    & \quad \quad w_{t+1}(i) \leftarrow w_t(i) \\
+    & \quad \quad \quad \quad - \eta \frac{g_t(i)}{\sqrt{\nu_t(i)}} \\
+    & \quad \quad \text{with the convention that } 0/0 = 0
+    \end{align*}
+
+
+
+  SM3-II Algorithm
+
+  The algorithm initializes with parameters such as the learning rate 
```

Comment:
Explains the difference with the SM3-I algorithm rather than rewriting a similar description